projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -841,8 +841,7 @@ private void checkQuestionsDirExists() {
    }
 
    @Override
```

Comment:
Isn't this an Autocloseable method that should throw Exception? I didn't even realize you were allowed to change the throws declaration of an overridden method.